common/src/main/java/io/homeassistant/companion/android/common/data/integration/impl/IntegrationRepositoryImpl.kt

```diff
@@ -352,6 +354,17 @@ class IntegrationRepositoryImpl @Inject constructor(
         return localStorage.getStringSet(PREF_WEAR_HOME_FAVORITES) ?: setOf()
     }
 
+    override suspend fun getTileShortcuts(): List<String> {
+        val jsonArray = JSONArray(localStorage.getString(PREF_TILE_SHORTCUTS) ?: "[]")
+        return List(jsonArray.length()) {
+            jsonArray.getString(it)
+        }
+    }
+
+    override suspend fun setTileShortcuts(entities: List<String>) {
+        localStorage.putString(PREF_TILE_SHORTCUTS, JSONArray(entities).toString())
+    }
+
```

Comment:
Why are we storing as a `String` if it's a list?

Reply:
For localStorage, the only other option is to store it as a map, but then we loose the order of the list

Reply:
Ok.... but only because we will be removing this in favor of the DB calls long term.

Reply:
If you see another option, I'm happy to change it now as well :)
This was the best I could come up with